test.sh

```diff
@@ -407,9 +418,38 @@ function run-all-client-server-perf-tests()
         NumClients=$3
     fi
 
+    set +x
+    local outfile="/tmp/${Me}.${FUNCNAME[0]}.out"
+    echo "${Me} tail -f $outfile"
+    echo " "
+    if [ -f "${outfile}" ]; then rm -rf "${outfile}"; fi
+
+    # Reset global, so it will be picked up when server binary is started.
+    SvrPerfOutfileArg="--perf-outfile ${outfile}"
```

Comment:
The global that is set here is picked up on L690 below, where server binary is run.